.env.development.local

```diff
@@ -1,3 +1,2 @@
-FAST_REFRESH=true
```

Comment:
We don't need this anymore?

Reply:
Yes, it's now enabled by default.
https://github.com/facebook/create-react-app/pull/9918